### DISCUSSION TITLE
Contributor Call: 7 January 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3201] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 7 January 2023 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jan-7-2023/5pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Possibility moving translations into patterns (See #3313 & #3310 )
- Possibility of adding a designs category (on freesewing.org) for new/beta designs, or for deprecated or so on
- Visibility/community/social platforms

> Notes will be added after the call.

**Comment by BenJamesBen** [comment]
- Add a designs category for new designs still in development?

So designers and customers get the benefits of having not-yet-complete designs added to and accessible from the repository and websites. And, there is less pressure to have designs/documentation be perfect before being added. And, customers' expectations are better managed if they come across and try these designs.

Possibly similar categories or tags for other types of designs? Experimental, Deprecated, Historical, Orphaned.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark
- @lexander2002
- @KaerMorhan
- @joostdecock
- @bobgeorgethe3rd
- @BenJamesBen
- @SeaZeeZee
- @woutervdub 
- @starfetch 
- @eriese 
- @anna-puk 
- @darigovresearch 

✨ **Benjamin & Joost**
Moving package-specific translations into packages (#3313).
Turn failing patterns into degraded patterns with warning messages (#3310).
There are messages that show up when the pattern is generated in a degraded version instead of not generating at all. They are not translated at the moment because we don’t have the infrastructure for it. A package is a unit we use for our code and translation is a separate package. Sometimes it is unclear if the text is still used and if we still need the translation. Moving translations closer to the original source will help keep that in order.

> _Joost: The risk is that it might scare people away from making patterns if they will also need to make translation packages._
>
> _Enoch: The thing I struggled a lot when I just joined FS is I didn’t know where to put the strings for them to be translated._
>
> _Wouter: I still don’t know that, that’s why I don’t do documentation._
>
> _Joost: This will help us to keep text and translations closer to the source._
>
> _Curvalicia: Will it be more complicated for translations?_
>
> _Joost: No, the translators will work the same on Crowding._

> _Benjamin: I will start writing documentation so the work will be in the process._

The possibility of adding a category for new designs that are still under development, experimental designs and so on.
It would be nice to have a low-risk category for people to contribute without the responsibility of testing, making it clean and maintaining it in the future.
So maybe people will have something in development that they would want to share or a historical pattern that is not that good but could be on the website for educational purposes.

> _Joost: One of the ways is to put patterns with a beta tag or category. The question is who will maintain this category? If we can leave the maintaining-ship out of the equation, we can just use tags and the new control setting (not showing the user those patterns if they want to keep it easy)._
>
> _Karen: I misunderstood the category in the beginning so maybe my misunderstanding could be a positive solution. There are a lot of patterns that are just in progress and maybe they would want to share before writing a complete pattern._
>
> _Bob: The question is how we define the beta section. Is it for the patterns that are for people who are having some trouble designing but want to develop it and maintain it later? Because we could to filter those beta patterns based on what the intentions of the author are._
>
> _Joost: V2 is loading now only published patterns from a separate package. V3 will load patterns directly from monorepo so every created new pattern can show up. It doesn’t mean that everything is visible but it can be._
>
> _Zee and Eriese: Some people don’t want to finish and publish patterns. It is a good idea to collaborate with other people on Discord to finish it._
>
> _Joost: Also, just to be clear, we do not gatekeep patterns that are not perfect._
>
> _Joost: I’m more of a fan of tags than categories and I’m not opposed to posting some of the beta patterns but we need to avoid new users or beginners getting confused by half-cooked patterns._
>
> _Bob: Maybe we can make beta a separate category which will also have tags inside of it._
>
> _Joost: I don’t know how to implement that yet but I also had an idea of letting beginners to see which patterns are more popular._

Discussion about FreeSewing visibility is in the [community](https://discord.com/channels/698854858052075530/757631165531291771) channel on DIscord.

**Comment by joostdecock** [comment]
Closing to give room to active discussions
